cura/CrashHandler.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, exception_type, value, tb):
         self.exception_type = exception_type
         self.value = value
         self.traceback = tb
+        self.dialog = QDialog()
```

Comment:
So the creation of the dialog should be removed from the _createDialog method, isn't it?